### DISCUSSION TITLE
Technical issues with camera & document capture

### DISCUSSION THREAD

**Discussion by Vanshbordia** [community/community#128305] [GitHub Education]
### Hi everyone,

These issues are there for a long time and no specific response to these issues

On desktop: firstly there is no direct way to choose camera, changed by browser settings, even though the camera is outputting in 1080p the captured image is always smaller than 650px in width and heights seems to be varying. This causes the image to blur and text not legible. 
OCR for text recognition also does not seem to work properly, official I'd card, transcript, admission letter all were disregard even though the university name is clearly mentioned, as well as the date and my name (same details as on my profile and billing)

Mobile: all phones are not able to change camera and capture image, (safari, chrome, edge, samsung browser, brave, firefox) no browser is letting change the camera and take the photo. Using front camera to focus, aim at the document is practically not possible. 

Bottom line even if the images are uploaded that are high quality and text clearly visible & following everything from the requirements, if the data is not in very specific format /place it's not accepting 

Fixes

- Document upload
- better camera library
- image upload sanitization, based 64 png is able to cause issues while upload and image is changeable before submitting 
- phone camera issue 
- format optimization for other universities & College

**Comment by paudelsamir** [comment]
It seems like a widespread issue. Have you tried using a third-party app for document capture, or maybe a different browser? GitHub Support might also be able to help with these specific problems.
